### ISSUE TITLE
GitHub Checks API

### ISSUE THREAD

**Issue by seniorquico** [Shippable/support#4322]
GitHub publicly announced today a new Checks API that appears to be replacing the commit statues from some other CI platforms (Travis CI, CircleCI, and App Center):

https://blog.github.com/2018-05-07-introducing-checks-api/
https://developer.github.com/changes/2018-05-07-new-checks-api-public-beta/

I'm interested to know if Shippable has been in touch with GitHub about this feature and whether or not an enhanced developer workflow is planned.

Thanks,

Kyle

**Comment by manishas** [comment]
Yes, this is already on our radar. We’ll update this issue when we have news to share.

**Comment by gundalow** [comment]
Hi,
Any update on this?

**Comment by gundalow** [comment]
@manishas Any updates you could share?

**Comment by JJ** [comment]
This is still the same, right?
